students/marijapavic/module-2-html/exercises/02-book/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>A book about markup languages</title>
+    </head>
+    <body>
+        <h1>1. Markup languages</h1>
+        <p>A <i>markup language</i> is a computer language that uses tags to define elements within a document. Markup languages are used to annotate a document in a way that is syntactically distinguishable from the text, and can assign semantic meaning to the text.</p>
```

Comment:
`<i>` prema HTML5 više ne koristimo jer nema semantičko značenje. Njegov efekt je samo vizualne prirode, a o tome ne brine HTML nego CSS. Ako želimo nešto istaknuti u semantičkom smislu možemo koristiti `<strong>` ili `<em>`. Progledat ću ti kroz prste ovdje 😄